src/main/java/sjchat/users/UserServiceServer.java

```diff
@@ -0,0 +1,89 @@
+package sjchat.users;
+
+import java.io.IOException;
+import java.util.Random;
+
+import io.grpc.Server;
+import io.grpc.ServerBuilder;
+import io.grpc.stub.StreamObserver;
+
+public class UserServiceServer {
+  private Server server;
+  private int port = 50051;
+
+  public static void main(String[] args) throws Exception {
+    UserServiceServer userServiceServer = new UserServiceServer();
+    userServiceServer.start();
+    userServiceServer.blockUntilShutdown();
+  }
+
+  private void start() throws IOException {
```

Comment:
All this seems like a very boiler-plate:y way of wrapping a gRPC service. Perhaps this should be extracted into the start of a service framework?